__tests__/Unit/Components/Issues/Card.test.tsx

```diff
@@ -8,11 +8,13 @@ const DEFAULT_PROPS: IssueItem = {
     title: 'Status view to all features being built for our app',
     user: {
         login: 'ankushdharkar',
+        html_url: 'https://github.com/ankushdharkar',
```

Comment:
please use dummy names

Reply:
It is necessary? btw it was already written.

Reply:
It is a good practice. Later on, someone can take action (even legally) against us for using their names without written consent.

Reply:
Done